BetSystem/BusinnessLogic/TeamService.cs

```diff
@@ -47,24 +46,24 @@ public TeamDto GetTeamById(int id)
             var team = _dbContext.Teams.FirstOrDefault(x => x.Id == id);
             if (team == null) return null;
             return new TeamDto
-            { Id = team.Id,
-              Name = team.Name };
-
+            {
+                Id = team.Id,
+                Name = team.Name
+            };
         }
 
-        public void  DeleteTeamById(int id)
+        public void DeleteTeamById(int id)
         {
-            var team = _dbContext.Teams.FirstOrDefault(x=>x.Id == id);
+            var team = _dbContext.Teams.FirstOrDefault(x => x.Id == id);
```

Comment:
Will it cascade delete remove all events/bets? Do we need to perform something that?

Reply:
No